install-ansible-ubuntu.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+sudo apt install software-properties-common
+sudo add-apt-repository --yes --update ppa:ansible/ansible
+sudo apt install ansible
```

Comment:
```suggestion
sudo apt install ansible

```

Reply:
```suggestion
sudo apt install ansible
ansible-playbook install-linux-ubuntu.yml --ask-become-pass
```
O que acha de já colocar a execução do playbook dentro do `.sh` e renomear esse arquivo para algo que referencie que vai instalar as aplicações na máquina windows ao invés de que vai instalar o ansible?

Reply:
Não é uma boa porque vai sempre ficar tentando adicionar o repositório e instalar.